### ISSUE TITLE
Some games require account?

### ISSUE THREAD

**Issue by OSVSL** [zach-morris/plugin.program.iagl#112]
Trying to launch some Saturn games. Says I need an account in settings? I figured it meant the archive.org account but I did that and it didn't work.

**Comment by zach-morris** [comment]
Thats exactly what it means.  Your login credentials are you archive.org user email and password.

**Comment by OSVSL** [comment]
Yea like I said I made an account and used it in the IAGL setup but the games still didn't work. Didn't know if this was an issue with archive.org, IAGL or the games themselves; just thought I 'd ask.

Awesome job with the add-on BTW absolutely loving it.

**Comment by zach-morris** [comment]
I tested dreamcast and it does work.  The error is vague and doesn't tell you specifically what the issue is.  Generally, archive.org returned an error screen rather than the requested file.  If you turn on debug logging, you should be able to see the requested link and try it in a web browser.  It's possible one specific game file is corrupt or something.

**Comment by OSVSL** [comment]
The games I tried specifically were Alien Trilogy and Doom, both gave me the same message that an account. I also (at random) tried Darius II and Gex and got the same error: Archive requires account email and password in settings. This has been done yet still no access. I will also add that I'm running Retroarch (kronos) as the external launcher. Not sure if that has anything to do with the issue, just thought I'd add that.
![Screenshot 2019-08-15 18 36 45](https://user-images.githubusercontent.com/54122718/63131516-fd260f00-bf8b-11e9-9840-81a7f4c97acf.png)
![Screenshot 2019-08-15 18 30 47](https://user-images.githubusercontent.com/54122718/63131517-fd260f00-bf8b-11e9-9cdb-446dcd12d1d9.png)

**Comment by zach-morris** [comment]
I tried Gex and Alien Trilogy.  They both worked, so I'm guessing they all do, and your either not connected to the internet, or your login is wrong somehow.  Like I said, you can look in your Kodi log and see the URL for the file and paste that into a web browser to see what you get.
<img width="600" src="https://i.imgur.com/m3yY5BF.png">

**Comment by OSVSL** [comment]
Okay Ill keep working with it and see whats happening. Thanks!

**Comment by zach-morris** [comment]
Closing issue

**Comment by lcampos42** [comment]
I have the same issue. But only with Dreamcast. 
I have entered in my logins for archive.org, checked my internet. And it still gives me the same error where it says archive requires account email and password in settings .
I have followed your instructions on where to enter the email and password it it shows up in there. I have erased the password and reentered it about 20+ times. I even created a new account on archive.org and tried new logins and I still get the same error message. 
Saturn games download fine for me, it is only a problem when I try Dreamcast games.
